include/aos/common/tools/error.hpp

```diff
@@ -9,6 +9,7 @@
 #define AOS_ERROR_HPP_
 
 #include <errno.h>
+#include <stdio.h>
```

Comment:
Move these changes to the separate commit.

Reply:
Done